docker/zap_common.py

```diff
@@ -510,8 +524,14 @@ def write_report(file_path, report):
 
         f.write(report)
 
+'''
+zap_import_context imports the context and stores a context_id which will be used by zap_active_scan. Therefore the
+context_id must be global so that zap_active_scan has access to the variable.
+'''
+
 @hook(wrap=True)
 def zap_import_context(zap, context_file):
+    global context_id
```

Comment:
The ID is being returned so the caller can pass the ID when calling the other function.